### PR TITLE
DISTPG-180 Contributing guide

### PR DESCRIPTION
**Pull request by nastena1606**
Added a contributing guide
Added a code of conduct

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+#### Install Sphinx and build locally
+
+1. Install [pip](https://pip.pypa.io/en/stable/installing/)
+2. Install [Sphinx](https://www.sphinx-doc.org/en/master/usage/installation.html).
+3. While in the root directory of the doc project, run the following command to build the documentation:
+
+```sh
+make clean html
+```
+4. Go to the ``build/html`` directory and open the ``index.html`` file to see the documentation.
```
Comment (LenzGr) [inline CONTRIBUTING.md]:
```suggestion
4. Go to the ``build/html`` directory and open the ``index.html`` file in your web browser to see the documentation.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
+# Contributing Guide
+
+Thank you for deciding to contribute and help us improve Percona Distribution for PostgreSQL documentation!
+
+We welcome contributors from all users and community. By contributing, you agree to the [Percona Community code of conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md).
```
Comment (LenzGr) [inline CONTRIBUTING.md:5]: Just wondering: if we're pointing to a central CoC that applies to all Percona projects here, is there a point in including a (potentially obsolete) copy of it in this repo?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+	- Sign in (create a Jira account if you don’t have one) and click **Create** to create an issue.
+	- Describe the issue you have detected in the Summary, Description, Steps To Reproduce, Affects Version fields.
+
+2. **[Contribute to documentation yourself](#contribute-to-documentation-yourself)**. There is the **Edit this page** link that leads you to the source file of the page on GitHub. There you make changes, create a pull request that we review and add to the doc project. For details how to do it, read on.
+
+![PPG links](source/_static/images/PPG_links.png)
+
+## Contribute to documentation yourself
+
+To contribute to documentation, learn about the following:
```
Comment (LenzGr) [inline CONTRIBUTING.md]:
```suggestion
To contribute to the documentation, you should be familiar with the following technologies:
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+- [reStructured text](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) markup language. It is used to write the documentation.
+- [Sphinx](https://www.sphinx-doc.org/en/master/usage/quickstart.html) documentation generator. We use it to convert source ``.rst`` files to html and PDF documents.
+- [git](https://git-scm.com/)
```
Comment (LenzGr) [inline CONTRIBUTING.md]: And github as well?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+- [Docker](https://docs.docker.com/get-docker/). It allows you to run Sphinx in a virtual environment instead of installing it and its dependencies on your machine.
+
+There are several active versions of the documentation. Each version has a branch in the repository named accordingly:
```
Comment (LenzGr) [inline CONTRIBUTING.md]: Would it make sense to explain that the versions are derived from the major version of PostgreSQL included in the distribution?

File: CONTRIBUTING.md (hunk 2 above)
Comment (nastena1606) [reply CONTRIBUTING.md:5]: Yes, you are absolutely right. I first included this doc before the link to it in the community repo was provided. Of course, it makes no sense keeping it here, will remove it.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+	 - In the *Commit changes* section, describe your changes.
+	 - Select the **Create a new branch for this commit and start a pull request** option
+	 - Click **Propose changes**.
+
+4. GitHub creates a branch and a commit for your changes. It loads a new page on which you can open a pull request to Percona. The page shows the base branch - the one you offer your changes for, your commit message and a diff - a visual representation of your changes against the original page.  This allows you to make a last-minute review. When you are ready, click the **Create pull request** button.
+5. Someone from our team reviews the pull request and if everything is correct, merges it into the documentation. Then it gets published on the site.
+
+### Edit documentation locally
+
+This option is for users who prefer to work from their computer and / or have the full control over the documentation process.
```
Comment (LenzGr) [inline CONTRIBUTING.md]:
```suggestion
This option is for users who prefer to work on a local copy of the documentation files and / or have the full control over the documentation process.
```

**Review by LenzGr — APPROVED** [review]
LGTM! Thank you.

**Pull request merged**